scene/resources/curve.cpp

```diff
@@ -2321,6 +2377,8 @@ void Curve3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_get_data"), &Curve3D::_get_data);
 	ClassDB::bind_method(D_METHOD("_set_data", "data"), &Curve3D::_set_data);
 
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "_closed"), "set_closed", "is_closed");
```

Comment:
```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "closed"), "set_closed", "is_closed");
```
This should not be prefixed with `_` unless you intend it to be hidden in the documentation, which I don't think should be done here